client/stylesheets/_modal.scss

```diff
@@ -467,7 +467,7 @@
 
 /*~~~~ Button ~~~~*/
 
-.modEditListingTrigger {
+.modEditListingTrigger, .modReportTrigger {
```

Comment:
Selector `modEditListingTrigger` should be written in lowercase with hyphens<br>Selector `modReportTrigger` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line